src/validation.cpp

```diff
@@ -3800,7 +3849,11 @@ void BlockManager::PruneOneBlockFile(const int fileNumber)
     setDirtyFileInfo.insert(fileNumber);
 }
 
-void BlockManager::FindFilesToPruneManual(std::set<int>& setFilesToPrune, int nManualPruneHeight, int chain_tip_height)
+void BlockManager::FindFilesToPruneManual(
+    std::set<int>& setFilesToPrune,
+    int nManualPruneHeight,
+    int chain_tip_height,
+    ChainstateManager& chainman)
```

Comment:
Sorry for causing the conflict, but BlockManager moved to blockstorage.cpp

Also, what is the status here? Do you plan to split another piece off or will this go in as one cake now?